### PR TITLE
Fetch data from api

### PR DESCRIPTION
**Pull request by DanSchrader**
Fetched from api and rendered a history chart as well as a current value table.

### CODE REVIEW

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -1,44 +1,83 @@
...
+  useEffect(() => {
+    const fetchHistoricalData = async () => {
+      let historicalDataArray = [];
+      await Axios.get(
+        'https://api.coingecko.com/api/v3/coins/raptoreum/market_chart?vs_currency=eur&days=27&interval=daily'
+      ).then((response) => {
+        historicalDataArray = response.data.prices;
+      });
+      let formattedData = formatFetchData(historicalDataArray);
+      console.log('formattedData', formattedData);
```
Comment (julialoeschel) [inline src/components/Dashboard.js:29]: I guess you dont need this anymore :)

File: src/components/Dashboard.js [hunk 2]
```diff
@@ -1,44 +1,83 @@
...
+        'https://api.coingecko.com/api/v3/coins/raptoreum?localization=de&tickers=false&market_data=true&developer_data=false&sparkline=false'
+      ).then((response) => {
+        currentDataArray = response.data;
+      });
+      setCoin(currentDataArray);
+    };
+
+    fetchCurrentData();
+  }, []);
+  console.log('coin', coin);
```
Comment (julialoeschel) [inline src/components/Dashboard.js:50]: same here - no more logging :)

**Review by julialoeschel — COMMENTED** [review]
Great, you made it work 💪 . 
I only found some console.logs .
keep up the good work!!!

File: src/components/Dashboard.js [hunk 3]
```diff
@@ -1,44 +1,83 @@
...
+          <TableIndicator>Marktkapazität:</TableIndicator>
+          <TableValue>{coin.market_data.market_cap.eur} EUR</TableValue>
+          <TableIndicator>Rang (Marktkapazität):</TableIndicator>
+          <TableValue>{coin.market_data.market_cap_rank}</TableValue>
+          <TableIndicator>Maximalmenge:</TableIndicator>
+          <TableValue>{coin.market_data.max_supply} RTM</TableValue>
+        </DashboardTableContainer>
+      ) : (
+        <div>Loading...</div>
+      )}
```
Comment (macfrei) [inline src/components/Dashboard.js:75]: It is a bit unusual to see three `Loading` texts at the screen. Maybe rework the conditional rendering to this: Or something like that 😅 
```suggestion
      {coin.length === 0 || historyData === 0  && <div>Loading...</div>}
      {historyData.length > 0 && <HistoryChart historyData={historyData} />}
      {coin.length > 0 && (
        <DashboardTableContainer>
          <TableIndicator>Preis:</TableIndicator>
          <TableValue>{coin.market_data.current_price.eur} EUR</TableValue>
          <TableIndicator>Marktkapazität:</TableIndicator>
          <TableValue>{coin.market_data.market_cap.eur} EUR</TableValue>
          <TableIndicator>Rang (Marktkapazität):</TableIndicator>
          <TableValue>{coin.market_data.market_cap_rank}</TableValue>
          <TableIndicator>Maximalmenge:</TableIndicator>
          <TableValue>{coin.market_data.max_supply} RTM</TableValue>
        </DashboardTableContainer>
      )}
```

**Pull request merged**